docs/conf/eu/2017/news/announcing-presentations-ticket-update.rst

```diff
@@ -0,0 +1,70 @@
+
+* Ed Stenson - `One-on-One 101 </conf/eu/2017/speakers/#speaker-ed-stenson>`_
+* Floriana Pagano - `Managing community-driven documentation - how to herd doc-writing cats through motivation and a pinch of automation </conf/eu/2017/speakers/#speaker-floriana-pagano>`_
+* Kathleen De Roo - `Manage your docs like an archivist </conf/eu/2017/speakers/#speaker-kathleen-de-roo>`_
+
+Important Ticket Pricing Update
+-------------------------------
+
+**The short version**: Based on community feedback, we reduced our ticket prices so that they are more accessible and inclusive to our European community, and refunded the difference to those who already purchased tickets.
+
+The new prices are:
+
+* 250 EUR for Corporate tickets (previously 300)
+* 150 EUR for Independent tickets (previously 200)
+* 75 EUR for Student & Unemployed tickets (previously 100)
+
+You can check out the detailed ticket pricing on our `Tickets <http://www.writethedocs.org/conf/eu/2017/tickets/>`_ page.
+
+**The long version**: Each year we examine our ticket pricing structure and determine the different tiers based on factors like production costs, attendee experience, and sponsorship funds offset. Our goal has always been to maintain a low price point and therefore a lower entry barrier to our events, so that documentarians from all walks of life can attend. As our community grows, our costs both inside and outside the conference grow too, and we have been gradually raising prices accordingly.
+
+In particular, we have changed how we budget our conferences to allow for paying some of our production staff and keeping a small sum to reinvest in operations outside of the conference, such as meetups and infrastructure. All this has added some overhead to the actual cost of the conference, but in hindsight, we should have maintained a more gradual increase in our ticket prices.
+
+After we received some candid feedback from the community, we re-examined the pricing structure and realized that we simply raised the prices too drastically this year. For example, the 2017 price for Independent tickets was twice as much as 2016. This is something we didn't catch at first, and doesn't reflect our values of accessibility and inclusion.
+
+Our events are not-for-profit, and it's the engagement of the documentarians in our community is what keeps the family growing. We are extremely grateful to our fellow documentarians who stepped up and communicated this feedback gracefully and who helped us through this process.
+
+We made a mistake, and we are fixing it. Standing by a mistake can be more damaging than owning up to it and fixing it. So we are fixing this mistake and lowering our prices.
+
+We hope that the new pricing structure better reflects our core values of accessibility and inclusivity. We already processed partial refunds for the price difference for those who already registered, but it might take a few days for the bank or credit card transactions.
+
+See you all in Prague!
```

Comment:
And maybe 

Looking forward to seeing you all in Prague, for talks, discussions, good food and of course, the boat tour!

Just so it isn't kthnxbai